.circleci/config.yml

```diff
@@ -6,6 +6,8 @@ workflows:
     jobs:
       - cypress/run:
           start: npm start
+          spec:
+            - "cypress/integration/index.test.js"
```

Comment:
We haven't put this in quotes in the past. Should we have?

Reply:
Oh I don't think it matters, I can just remove them